docs/DevelopersGuide.md

```diff
@@ -77,12 +77,34 @@ Please see the following document for `configure` usage: [BuildOptions.md](./Bui
 
 The build output is in the `_build` directory. The following applications can be found there.
 
-* `srt-live-transmit` — A sample application to transmit a live stream from source medium (UDP/SRT/`stdin`)
+* `srt-live-transmit` - A sample application to transmit a live stream from source medium (UDP/SRT/`stdin`)
 to the target medium (UDP/SRT/`stdout`). See [srt-live-transmit.md](./srt-live-transmit.md) for more info.
-* `srt-file-transmit` — A sample application to transmit files with SRT.
-* `srt-tunnel` — A sample application to set up an SRT tunnel for TCP traffic. See [srt-tunnel.md](./srt-tunnel.md) for more info.
+* `srt-file-transmit` - A sample application to transmit files with SRT.
+* `srt-tunnel` - A sample application to set up an SRT tunnel for TCP traffic. See [srt-tunnel.md](./srt-tunnel.md) for more info.
 * `tests-srt` - unit testing application.
 
+## Language standard requirements
+
+The following conventions for the language standard are used in this project:
+
+1. The SRT library requires C++03 (also known as C++98) standard.
+2. The examples (to be enabled by `--enable-examples`) require either C++03 or C89 standard.
+3. The following require C++11 standard:
+   * demo applications
+   * testing applications (to be enabled by `--enable-testing`)
+   * unit tests (to be enabled by `--enable-unittests`)
+
+Note that C++11 standard will be enforced if you have enabled applications
+and haven't specified the C++ standard explicitly. When you have an old compiler
+that does not support C++11 and you want to compile as many parts as possible,
+the simplest way is to use the following options:
+
+```
+./configure --disable-apps --use-c++-std=03 --enable-examples
+```
```

Comment:
CMake should be the default build system. TCL is not available on Windows by default.
```
cmake . -DENABLE_APPS=ON and so on
```